main/modes/games/cosCrunch/microgameA.c

```diff
@@ -0,0 +1,96 @@
+#include "microgameA.h"
+#include "cosCrunch.h"
+
+static void microgameAInitMicrogame(void);
+static void microgameADestroyMicrogame(void);
+static void microgameAMainLoop(int64_t elapsedUs, uint64_t timeRemainingUs, cosCrunchMicrogameState state,
+                               buttonEvt_t buttonEvts[], uint8_t buttonEvtCount);
+
+static const char microgameAVerb[] = "Just Press \"A\"!";
+cosCrunchMicrogame_t ccMicrogameA  = {
+     .verb               = microgameAVerb,
+     .timeoutSeconds     = 3,
+     .fnInitMicrogame    = microgameAInitMicrogame,
+     .fnDestroyMicrogame = microgameADestroyMicrogame,
+     .fnMainLoop         = microgameAMainLoop,
+};
+
+typedef struct
+{
+    font_t font;
+    cosCrunchMicrogameState previousState;
+    uint64_t timeSinceTextMovedUs;
+    uint16_t textX;
+    uint16_t textY;
+} microgameA_t;
+microgameA_t* microgameA = NULL;
+
+#define TEXT_MOVE_DELAY_US 600000
+
+static void microgameAInitMicrogame(void)
+{
+    microgameA = heap_caps_calloc(1, sizeof(microgameA_t), MALLOC_CAP_8BIT);
+    loadFont(IBM_VGA_8_FONT, &microgameA->font, false);
```

Comment:
This font is getting double-loaded, which we do have the RAM for, but is inefficient. One option is to let the microgames use the fonts allocated by the mode. Another option is to do #418 and be smarter about loading (@dylwhich, look, a use case!). Food for thought.

Reply:
I don't think it's unreasonable for the main mode to load all common assets, like fonts, and supply them to the microgames. Makes it easier to be at least a little cohesive.

A list to load is silly.